Update quick start - mention how to start the web ui (#263)
google/wasefire: book/src/quick/README.md

Depending on your hardware, you can run the hello applet with:

> - `cargo xtask applet rust hello runner nordic` to run on an nRF52840 dev-kit

Reviewer: ```suggestion
- `cargo xtask applet rust hello runner nordic` to run on an nRF52840 dev-kit.
```

- `cargo xtask applet rust hello runner host` to run on your desktop. 

>    Add `--features=web` to start a web UI for your board.

Reviewer: ```suggestion
- `cargo xtask applet rust hello runner host --web` to run on your desktop with a web UI.
```
